packages/ses/src/error/types.js

```diff
@@ -341,6 +341,7 @@
  *   details: DetailsTag,
  *   Fail: FailTag,
  *   quote: AssertQuote,
+ *   raw: AssertQuote,
```

Comment:
My only remaining requested change is to document the need to avoid using `raw`/`bare` on possibly attacker controlled data, lest it give the attacker the opportunity to introduce a quoting confusion in the interpretation of the error string. I leave it to you to determine where this explanation should go.

Reply:
Done; please re-review.